{"primaryContentSections":[{"kind":"declarations","declarations":[{"tokens":[{"kind":"keyword","text":"case"},{"kind":"text","text":" "},{"kind":"identifier","text":"unknown"}],"languages":["swift"],"platforms":["iOS"]}]}],"schemaVersion":{"major":0,"minor":3,"patch":0},"sections":[],"variants":[{"paths":["\/documentation\/apptentivekit\/apptentiveauthenticationfailurereason\/unknown"],"traits":[{"interfaceLanguage":"swift"}]}],"identifier":{"url":"doc:\/\/com.apptentive.Apptentive\/documentation\/ApptentiveKit\/ApptentiveAuthenticationFailureReason\/unknown","interfaceLanguage":"swift"},"abstract":[{"type":"text","text":"An unknown authentication failure."}],"kind":"symbol","metadata":{"role":"symbol","title":"ApptentiveAuthenticationFailureReason.unknown","roleHeading":"Case","fragments":[{"kind":"keyword","text":"case"},{"kind":"text","text":" "},{"kind":"identifier","text":"unknown"}],"symbolKind":"case","externalID":"c:@M@ApptentiveKit@E@ApptentiveAuthenticationFailureReason@ApptentiveAuthenticationFailureReasonUnknown","modules":[{"name":"ApptentiveKit"}],"platforms":[]},"hierarchy":{"paths":[["doc:\/\/com.apptentive.Apptentive\/documentation\/ApptentiveKit","doc:\/\/com.apptentive.Apptentive\/documentation\/ApptentiveKit\/ApptentiveAuthenticationFailureReason"]]},"deprecationSummary":[{"type":"paragraph","inlineContent":[{"type":"text","text":"Multiple users on the same device is not currently supported."}]}],"references":{"doc://com.apptentive.Apptentive/documentation/ApptentiveKit/ApptentiveAuthenticationFailureReason":{"deprecated":true,"role":"symbol","title":"ApptentiveAuthenticationFailureReason","fragments":[{"kind":"keyword","text":"enum"},{"kind":"text","text":" "},{"kind":"identifier","text":"ApptentiveAuthenticationFailureReason"}],"abstract":[],"identifier":"doc:\/\/com.apptentive.Apptentive\/documentation\/ApptentiveKit\/ApptentiveAuthenticationFailureReason","kind":"symbol","type":"topic","navigatorTitle":[{"kind":"identifier","text":"ApptentiveAuthenticationFailureReason"}],"url":"\/documentation\/apptentivekit\/apptentiveauthenticationfailurereason"},"doc://com.apptentive.Apptentive/documentation/ApptentiveKit":{"role":"collection","title":"ApptentiveKit","abstract":[],"identifier":"doc:\/\/com.apptentive.Apptentive\/documentation\/ApptentiveKit","kind":"symbol","type":"topic","url":"\/documentation\/apptentivekit"},"doc://com.apptentive.Apptentive/documentation/ApptentiveKit/ApptentiveAuthenticationFailureReason/unknown":{"role":"symbol","title":"ApptentiveAuthenticationFailureReason.unknown","fragments":[{"kind":"keyword","text":"case"},{"kind":"text","text":" "},{"kind":"identifier","text":"unknown"}],"abstract":[{"type":"text","text":"An unknown authentication failure."}],"identifier":"doc:\/\/com.apptentive.Apptentive\/documentation\/ApptentiveKit\/ApptentiveAuthenticationFailureReason\/unknown","kind":"symbol","type":"topic","deprecated":true,"url":"\/documentation\/apptentivekit\/apptentiveauthenticationfailurereason\/unknown"}}}